Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM php:7.3.14-fpm-alpine3.11
+FROM php:7.4.14-fpm-alpine3.12
 
 # Docker Build Arguments
-ARG RESTY_VERSION="1.15.8.2"
-ARG RESTY_OPENSSL_VERSION="1.0.2t"
-ARG RESTY_PCRE_VERSION="8.43"
-ARG REDIS_VERSION="5.0.2"
-ARG MEMCACHED_VERSION="3.1.4"
-ARG XDEBUG_VERSION="2.7.2"
+ARG RESTY_VERSION="1.15.8.3"
```

Comment:
did you decide not to go to OpenRESTY 1.17.x or 1.19.x b/c these added new features but no bugfix/security updates and there might be breakages?

Reply:
That more or less sums it up. 1.15.8.3 picks up the main security fixes and 17/19 appeared to differ primarily in added features which seemed like unnecessary risk of breakages.